core-plugins/src/main/java/co/cask/hydrator/plugin/batch/source/ExcelInputReader.java

```diff
@@ -493,6 +493,10 @@ public void validate() {
         throw new IllegalArgumentException("Value for Table Expiry period is valid only when file tracking table " +
                                              "is specified. Please specify file tracking table name.");
       }
+
+      if (!containsMacro("rowsLimit") && !StringUtils.isNumeric(rowsLimit)) {
```

Comment:
This might fail as isNumeric returns true for empty string based on what version of apache commons you are using.